jakarta-package-alignment-mappings/src/test/java/com/palantir/gradle/jakartapackagealignment/VersionMappingsTest.java

```diff
@@ -0,0 +1,33 @@
+/*
+ * (c) Copyright 2023 Palantir Technologies Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.palantir.gradle.jakartapackagealignment;
+
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import java.util.Optional;
+import org.junit.jupiter.api.Test;
+
+class VersionMappingsTest {
+    @Test
+    public void testBogusVersions() {
```

Comment:
unnecessary public modifier, probably worth a new error-prone rule+excavator to handle this sort of thing more broadly when I have some cycles.
```suggestion
   void testBogusVersions() {
```